chia/consensus/multiprocess_validation.py

```diff
@@ -304,8 +310,9 @@ async def pre_validate_blocks_multiprocessing(
             curr_b: FullBlock = block
 
             while curr_b.prev_header_hash in block_dict:
+                header_hash = curr_b.prev_header_hash
```

Comment:
Only a minor consideration: Perhaps it would be clearer if the `header_hash` var in this section would be named `prev_header_hash` instead, and then used in all 3 subsequent reads (as opposed to alternated with `curr_b.prev_header_hash` in the following line:
```python
prev_header_hash = curr_b.prev_header_hash
curr_b = block_dict[prev_header_hash]
prev_blocks_dict[prev_header_hash] = curr_b
```

Reply:
I chose `header_hash` because it's really only the "previous" hash in the first two lines, and the "current" header hash on the 3rd line. `header_hash` is a bit neutral on which one it is